src/components/BottomSheetBase.tsx

```diff
@@ -32,7 +32,12 @@ const BottomSheetBase = ({
 
   const renderBackdrop = useCallback(
     (props: BottomSheetDefaultBackdropProps) => (
-      <BottomSheetBackdrop {...props} disappearsOnIndex={-1} appearsOnIndex={0} />
+      <BottomSheetBackdrop
+        {...props}
+        style={[props.style, styles.bottomSheetBackdrop]}
```

Comment:
turns out we weren't setting the backdrop color before, the lib sets it to black by default which is what i've kept in the colors